src/vaishnavi_SeleniumBasics/VerifyFirstWebPage.java

```diff
@@ -0,0 +1,79 @@
+package vaishnavi_SeleniumBasics;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import jdk.management.resource.internal.inst.SocketOutputStreamRMHooks;
+
+public class VerifyFirstWebPage {
+
+	public static void main(String[] args) {
+		System.setProperty("webdriver.chrome.driver", ".\\resources\\windows//chromedriver.exe");
+		WebDriver driver = new ChromeDriver();
+		driver.get("file:///C:/Users/vaish/Desktop/Selenium_Recordings/TechnoCredits.html");
+
+		// Trying to verify title of the page
+		if (driver.getTitle().equals("TECHNOCREDITS"))
+			System.out.println("Yes, You are on a correct Page with title TECHNOCREDITS");
+		else
+			System.out.println("You are on a page of title: " + driver.getTitle());
+
+		// Filling all the details of form
+		driver.findElement(By.id("first name")).sendKeys("Vaishnavi");
+		driver.findElement(By.id("last name")).sendKeys("Vaidya");
+		driver.findElement(By.id("E-mail")).sendKeys("Vaishnavivaidya06@gmail.com");
+		driver.findElement(By.id("Company Name")).sendKeys("Atos-Syntel");
+
+		WebElement maleRadioButton = driver.findElement(By.id("male"));
+		maleRadioButton.click();
+
+		WebElement experience = driver.findElement(By.id("1-3"));
+		experience.click();
+		driver.findElement(By.name("lang")).sendKeys("JAVA");
+		WebElement language = driver.findElement(By.id("script"));
+		language.click();
+		WebElement dreamCompany = driver.findElement(By.id("google1"));
+		dreamCompany.click();
+		WebElement termsAndCondition = driver.findElement(By.id("check"));
+		termsAndCondition.click();
+
+		// Reseting the values of this form
+		driver.findElement(By.id("reset12")).click();
+
+		// Checking if all the radio buttons and check boxes reset is successfully done
+		if (maleRadioButton.isSelected() == false)
```

Comment:
Verify Reset works properly on your text field, radio button and checkbox.

here you missed validation for text field.

Reply:
-- Added one condition for text field as well

Reply:
-- Thank you for all the suggestions. :)